From: Matthias Schiffer <mschiffer@universe-factory.net>
Date: Sat, 21 Nov 2015 18:35:49 +0100
Subject: CC: base-files: add missing public key

Add the secondary package signature key to the 15.05 sources so that people
building the release have the same keys as people using precompiled images.

Signed-off-by: Jo-Philipp Wich <jow@openwrt.org>

diff --git a/package/base-files/files/etc/opkg/keys/de98a2dd1d0f8a07 b/package/base-files/files/etc/opkg/keys/de98a2dd1d0f8a07
new file mode 100644
index 0000000..5e3088c
--- /dev/null
+++ b/package/base-files/files/etc/opkg/keys/de98a2dd1d0f8a07
@@ -0,0 +1,2 @@
+untrusted comment: openwrt.org 15.05 release key
+RWTemKLdHQ+KBxOILy8gyk+5PaDVdfyJ32TFnY/jnQOrBAd1wobbLNYz
